bson.go

```diff
@@ -125,3 +125,19 @@ func (id *ObjectId) SetBSON(raw bson.Raw) error {
 
 	return errors.New("couldn't unmarshal bson raw value as ObjectId")
 }
+
+// DeepCopyInto copies the reciever and writes its value into out.
+func (in *ObjectId) DeepCopyInto(out *ObjectId) {
+	*out = *in
+	return
+}
+
+// DeepCopy copies the receiver into a new ObjectId.
+func (in *ObjectId) DeepCopy() *ObjectId {
+	if in == nil {
```

Comment:
Why can `in` be `nil` in `DeepCopy` but not `DeepCopyInto`?

Reply:
Or sorry, you are right... Please disregard the previous comment. I will add `nil` check for `DeepCopyInto` in a follow up PR.